.github/workflows/ci_suite.yml

```
@@ -89,6 +91,37 @@ jobs:
           source $HOME/BYOND/byond/bin/byondsetup
           tools/build/build --ci dm -DCIBUILDING -DCITESTING -DALL_MAPS
 
+  test_tgs_docker:
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    name: Test TGS Docker
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-20.04
    concurrency:
      group: test_tgs_docker-${{ github.head_ref || github.run_id }}
      cancel-in-progress: true
```

In case someone does rapid fire code-review-suggestion-application, it allows us to free up the runner on the old commit and await a run on the freshest commit https://docs.github.com/en/actions/using-jobs/using-concurrency

no warranty on YML fucking up if you apply my shoddy copy-paste as an actual commit